### DISCUSSION TITLE
error in docker-entrypoint.sh in version 2021.3.1

### DISCUSSION THREAD

**Discussion by fantomasdm** [INSPIRE-MIF/helpdesk-validator#781] [Ideas]
I think is to change line with unzip -o ui.zip -d "$ETF_DIR"
with unzip  ui.zip -d "$ETF_DIR"
otherwise everytime restart docker container etf folder is overwrite by ui.zip
Old version haven't -o parameter

**Comment by carlospzurita** [comment]
Dear @fantomasdm 

What is the error you are getting? The `-o` option was precisely to overwrite the UI folder and avoid clashes between file versions. Can you be more specific about the issue that you are having with this operation?

**Comment by fantomasdm** [comment]
If change static html on ~/etf/validator/home/index.html every time restart docker I lost my change.

**Comment by fabiovinci** [comment]
Dear @fantomasdm,

do you still have the problem? Have you tried the latest version?

**Comment by fantomasdm** [comment]
I still have same problem, but change docker-entrypoint.sh  in res  replacing [unzip -o ui.zip -d "$ETF_DIR"] with [unzip ui.zip -d "$ETF_DIR"] and rebuilding image and resolve! I need to custumize ui for traslate in italian.

**Comment by fabiovinci** [comment]
Dear @fantomasdm,

thank you for your feedback, it could be useful for other users that want to customize the UI.
For this reason, I will convert this issue into a discussion.

We would like to maintain the `-o` option to overwrite the UI folder and avoid clashes between file versions, but we will evaluate the opportunity to insert the command without the `-o` option as an alternative.
